tests/has_many_test.js

```diff
@@ -0,0 +1,1606 @@
+        expect(users[1].tasks[0].title).to.equal('a');
+        expect(users[1].tasks[0].subtasks.length).to.equal(3);
+        expect(users[1].tasks[0].subtasks[0].title).to.equal('c');
+        expect(users[1].tasks[0].subtasks[1].title).to.equal('b');
+        expect(users[1].tasks[0].subtasks[2].title).to.equal('a');
+
+        expect(users[1].tasks[1].title).to.equal('b');
+        expect(users[1].tasks[1].subtasks.length).to.equal(2);
+        expect(users[1].tasks[1].subtasks[0].title).to.equal('b');
+        expect(users[1].tasks[1].subtasks[1].title).to.equal('a');
+      });
+
+      // Reason: https://github.com/sequelize/sequelize/issues/13088
+      it.skip('should fetch associations for multiple instances with limit and order and a belongsTo relation', async function() {
+        const User = this.sequelize.define('User', {}),
+          Task = this.sequelize.define('Task', {
+            title: DataTypes.STRING,
+            categoryId: {
+              type: DataTypes.INTEGER,
+              field: 'category_id'
+            }
+          }),
+          Category = this.sequelize.define('Category', {});
+
+        User.Tasks = User.hasMany(Task, { as: 'tasks' });
+        Task.Category = Task.belongsTo(Category, { as: 'category', foreignKey: 'categoryId' });
+
+        await this.sequelize.sync({ force: true });
+
+        const users = await Promise.all([User.create({
+          tasks: [
+            { title: 'b', category: {} },
+            { title: 'd', category: {} },
+            { title: 'c', category: {} },
+            { title: 'a', category: {} }
+          ]
+        }, {
+          include: [{ association: User.Tasks, include: [Task.Category] }]
+        }), User.create({
+          tasks: [
+            { title: 'a', category: {} },
+            { title: 'c', category: {} },
+            { title: 'b', category: {} }
+          ]
+        }, {
+          include: [{ association: User.Tasks, include: [Task.Category] }]
+        })]);
+
+        const result = await User.Tasks.get(users, {
+          limit: 2,
+          order: [
+            ['title', 'ASC']
+          ],
+          include: [Task.Category]
+        });
+
+        expect(result[users[0].id].length).to.equal(2);
+        expect(result[users[0].id][0].title).to.equal('a');
+        expect(result[users[0].id][0].category).to.be.ok;
+        expect(result[users[0].id][1].title).to.equal('b');
+        expect(result[users[0].id][1].category).to.be.ok;
+
+        expect(result[users[1].id].length).to.equal(2);
+        expect(result[users[1].id][0].title).to.equal('a');
+        expect(result[users[1].id][0].category).to.be.ok;
+        expect(result[users[1].id][1].title).to.equal('b');
+        expect(result[users[1].id][1].category).to.be.ok;
+      });
+
+      // Reason: works correctly, but it fails because it tries to drop table crdb_internal from cockroach
+      it.skip('supports schemas', async function() {
```

Comment:
If I'm reading this test correctly, we're creating a new schema and dropping it right? This seems like it should pass.

What table is it trying to drop?